File: internal/content/testdata/bench/go/gas-station.go
func can_complete_circuit(gas []int, cost []int) int {
	totalGas, totalCost := 0, 0
	for i := range gas {
		totalGas += gas[i]
		totalCost += cost[i]
	}
	if totalGas < totalCost {
		return -1
	}
	total := 0
	start := 0
	for i := range gas {
		total += gas[i] - cost[i]
		if total < 0 {
			start = i + 1
			total = 0
		}
	}
	return start
}

File: internal/content/testdata/bench/javascript/gas-station.js
function can_complete_circuit(gas, cost) {
    const totalGas = gas.reduce((a, b) => a + b, 0);
    const totalCost = cost.reduce((a, b) => a + b, 0);
    if (totalGas < totalCost) return -1;
    let total = 0, start = 0;
    for (let i = 0; i < gas.length; i++) {
        total += gas[i] - cost[i];
        if (total < 0) {
            start = i + 1;
            total = 0;
        }
    }
    return start;
}
